src/redux/categories/categoriesSlide.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: (state) => ({
+      ...state,
+      categories: ['Under construction"'],
+    }),
+  },
+});
+
```

Comment:
- [ ] As with the booksSlice.js, you should also export `checkStatus` as an action.
`export const { checkStatus } = categoriesSlice.actions`